docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-datadog.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Access Requests with Datadog Incident Management
+description: How to set up Teleport's Datadog Incident Management plugin for privilege elevation approvals.
+---
+
+## How it works
+
+With Teleport's Datadog Incident Management integration, engineers can access the
+infrastructure they need to resolve incidents without longstanding admin
+permissions that can become a vector for attacks.
+
+Teleport's Datadog Incident Management integration allows you to treat Teleport
+Role Access Requests as Datadog incidents, notify the appropriate on-call team,
+and approve or deny the requests via Teleport.
+
+<Details title="This integration is hosted on Teleport Enterprise (Cloud)" open={false}>
```

Comment:
It's not clear if this is a cloud-only feature, enteprise-only, or can still be done by running the self-hosted plugin (we do have [the main function here](https://github.com/gravitational/teleport/blob/master/integrations/access/datadog/cmd/teleport-datadog/main.go) for the standalone executable).